.eslintrc.json

```diff
@@ -60,7 +60,8 @@
 		"import/prefer-default-export": "off",
 		"react/prop-types": "off",
 		"react/jsx-props-no-spreading": "off",
-		"react/require-default-props": "off"
+		"react/require-default-props": "off",
+		"no-underscore-dangle": "off"
```

Comment:
I think we should make this stricter, the point of using ESLint is to enforce style.